serverlessworkflow/modules/ROOT/pages/core/custom-functions-support.adoc

```diff
@@ -205,6 +205,120 @@ public class MyInterfaceOrClass {
+ When a Serverless workflow function is called, Kogito identifies the proper `WorkflowWorkItemHandler` instance to be used for that function type (using the handler name associated with that type by the deployment project) and then invokes the `internalExecute` method. The `Map` parameter contains the function arguments defined in the Workflow, and the `WorkItem` parameter contains the metadata information added to the handler by the deployment project. Hence, the `executeWorkItem` implementation has access to all the information needed to perform the computational logic intended for that custom type. 
+
+
+=== Custom function type example. 
+
+Assuming you want to interact, from a workflow file, with a legacy RPC server as the one defined link:{kogito_sw_examples_url}/serverless-workflow-custom-type/serverless-workflow-custom-rpc-server[this project]. This legacy server supports four simple arithmetic operations: add, minus, multiply and divide, which can be invoked using a custom RPC protocol. Since this an uncommon protocol, the workflow cannot handle them by using any of the predefined Serverless Workflow function types. The available options are to use a Java Service, which invokes a Java class that knows how to interact with the server; or define a custom type that knows how to interact with the service. 
+
+Using  the latter approach, users can write a link:{kogito_sw_examples_url}/serverless-workflow-custom-type/serverless-workflow-custom-type-example/src/main/resources/customType.sw.json[workflow file] defining this function. 
+
+.RPC Custom Function definition example
+[source,json]
+----
+ "functions": [
+    {
+      "name": "division",
+      "type": "custom",
+      "operation": "rpc:division"
+    }
+  ],
+----
+The `operation` starts with `rpc`, which is the custom type identifier, and continues with `division`, which denotes the operation that is going to be executed in the legacy server
+
+For this function definition to be recognized, a Kogito addon that defines the `rpc` custom type needs to be developed: it consists of a  link:{kogito_sw_examples_url}/serverless-workflow-custom-type/serverless-workflow-custom-rpc-deployment[deployment project] and a link:{kogito_sw_examples_url}/serverless-workflow-custom-type/serverless-workflow-custom-rpc[runtime project].
+
+The deployment project is responsible for extending link:{kogito_sw_examples_url}/serverless-workflow-custom-type/serverless-workflow-custom-rpc-deployment/src/main/java/org/kie/kogito/examples/sw/services/RPCCustomTypeHandler.java[`WorkItemTypeHandler`] and setup the `WorkItemNodeFactory`:
+
+.Example of the RPC Function Java implementation
+[source,java]
+----
+
+import static org.kie.kogito.examples.sw.custom.RPCCustomWorkItemHandler.NAME;
+import static org.kie.kogito.examples.sw.custom.RPCCustomWorkItemHandler.OPERATION;
+    @Override
+    protected <T extends RuleFlowNodeContainerFactory<T, ?>> WorkItemNodeFactory<T> fillWorkItemHandler(Workflow workflow,
```

Comment:
It would be also good to include the `type()` method here to show the mapping `type - WorkItemHandler`.